modules/cluster/README.md

```diff
@@ -0,0 +1,214 @@
+# AWS ECS Terraform module
```

Comment:
```suggestion
# AWS ECS Cluster Terraform module
```

Reply:
Updated in https://github.com/terraform-aws-modules/terraform-aws-ecs/pull/76/commits/c6fef3b1bb52c658ce3cd0a120a38c6e17bef170